universal-login-wallet/src/ui/react/Modals/Modal.tsx

```diff
@@ -51,13 +51,13 @@ const Modal = () => {
     case 'waitingForDeploy':
       return (
         <ModalWrapper modalClassName="jarvis-modal">
-          <ModalWaitingFor action={'Txn pending'} />
+          <ModalWaitingFor action={'Txn pending'} transactionHash={'0x5a63…f3a68a'}/>
```

Comment:
Magic value: `0x5a63…f3a68a`?

Reply:
It was hard coded into modal and then extracted into ModalParameters. It is placeholder for future real transactionHash from deployment

Reply:
ok